ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx

```diff
@@ -448,6 +453,8 @@ export const SmartTransactionStatusPage = ({
     dispatch(hideLoadingIndication());
   }, []);
 
+  const canShowSimulationDetails =
+    fullTxData.simulationData?.tokenBalanceChanges?.length > 0;
```

Comment:
For now yes, since showing it e.g. for the Send feature wasn't so useful. Let's sync up on this tomorrow and I can send another PR to allow more use cases if we want to.